Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-search.tsx

```diff
@@ -79,6 +87,38 @@ const SearchProvider = ({ children }: ISearchProviderProps) => {
         [ encodeUrlToUTF8, urlTerms ],
     );
 
+    useEffect(
+        () => {
+            if (isNil(getSearchResultsUrlParams)) {
+                return;
+            }
+
+            setIsSearchingUsers(false);
+            setIsSearchingProblems(false);
+            setIsSearchingContests(false);
+
+            const isProblemsCategoryInUrl = !isEmpty(getSearchResultsUrlParams?.selectedTerms
+                .filter(({ key }) => key === 'Problems'));
```

Comment:
Export 'Problems', 'Contests' and 'Users' aswell

Reply:
Fixed it there as well as in the urlTerms  and replaced it with SearchCategory enum